pkg/option/option.go

```diff
@@ -245,6 +276,11 @@ func (opt *Options) Parse() (string, error) {
 	}
 
 	opt.renameLegacyClusterRoles()
+
+	if opt.UseStandaloneEtcd {
+		opt.ClusterRole = "secondary"
```

Comment:
It could be more visible / explicit that when using "standalone" etcd, the cluster role cannot be "primary". Logging is not yet initialized here but maybe a comment:
```suggestion
		opt.ClusterRole = "secondary" // when using external standalone etcd, the cluster role cannot be "primary"
```

Reply:
updated